libnavigation-core/src/main/java/com/mapbox/navigation/core/accounts/Billing.kt

```diff
@@ -14,7 +14,7 @@ internal class Billing private constructor() {
     }
 
     companion object {
-        private var INSTANCE: Billing? = null
+        internal var INSTANCE: Billing? = null
```

Comment:
@Guardiola31337 - after our discussion, I feel like this is the easiest way to do it. I know you are uncomfortable with this way but given that the class and the variable are both declared as internal. Might be fine. What do you think?